modules/test/poshi/poshi-runner/src/main/java/com/liferay/poshi/runner/selenium/BaseWebDriverImpl.java

```diff
@@ -206,7 +206,7 @@ public void antCommand(String fileName, String target) throws Exception {
 		Future<Void> future = executorService.submit(antCommands);
 
 		try {
-			future.get(150, TimeUnit.SECONDS);
+			future.get(600, TimeUnit.SECONDS);
```

Comment:
CheckUpgradeClientUpgradeCoreOnly and CheckUpgradeClientGogoShellFailedUpgrade tend to stop before the upgrade process can finish. 

**Examples:** 
[CheckUpgradeClientUpgradeCoreOnly](https://testray.liferay.com/reports/production/logs/2022-12/test-1-27/test-portal-acceptance-pullrequest(master)/6066/functional-upgrade-tomcat90-mysql57-jdk8/0/12/jenkins-console.txt.gz?authuser=0) with 150 seconds. (doesn't finish upgrade process)
[CheckUpgradeClientUpgradeCoreOnly](https://testray.liferay.com/reports/production/logs/2022-12/test-1-39/test-portal-acceptance-pullrequest(master)/2084/functional-upgrade-tomcat90-mysql57-jdk8/0/12/jenkins-console.txt.gz?authuser=0) with 600 seconds. (finishes upgrade process and continues with the rest of the script)

Increasing the time seems to let it finish running before quitting. Not sure if this is something we should ask for or if there is a better way.

Reply:
Oh I guess this isn't full-proof because the upgrade process didn't finish in the latest test run https://testray.liferay.com/reports/production/logs/2022-12/test-1-37/test-portal-acceptance-pullrequest(master)/6748/functional-upgrade-tomcat90-mysql57-jdk8/0/12/jenkins-console.txt.gz?authuser=0